osquery/tables/other/tests/augeas_tests.cpp

```diff
@@ -0,0 +1,103 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+#include <gtest/gtest.h>
+
+#include <osquery/sql.h>
+
+namespace osquery {
+
+namespace tables {
+
+class AugeasTests : public testing::Test {};
+
+TEST_F(AugeasTests, select_all) {
+  auto results = SQL("select * from augeas");
+  ASSERT_TRUE(results.getStatus().ok()) << results.getStatus().what();
+  ASSERT_GT(results.rows().size(), 0U);
+}
+
+TEST_F(AugeasTests, select_all_with_limit) {
+  auto results = SQL("select * from augeas limit 5");
+  ASSERT_TRUE(results.getStatus().ok()) << results.getStatus().what();
+  ASSERT_EQ(results.rows().size(), 5U);
+}
```

Comment:
While this does test your table implicitly, it is a flaky test, let's drop this.